Dockerfile.plugin-runtime

```diff
@@ -0,0 +1,8 @@
+# This is the Docker image that Grapl plugins will run inside until we swap
+# over to Firecracker. 
+# It needs to be able to run Rust binaries (nothing special) and 
+# python PEXs (needs py3.10).
+
+ARG PYTHON_VERSION
+
+FROM python:${PYTHON_VERSION}-slim-bullseye AS docker-plugin-runtime
```

Comment:
A slim-bullseye image is a good starting point... do we have any notion of what other requirements will be needed for actually executing plugins beyond a Python runtime? It'd be nice to have as small a surface area as possible. But, as said in another comment, it's obviously not a huge deal at the moment.

Reply:
That's about it for now